docs/documentation/make-first-prototype/branching.md

```diff
@@ -0,0 +1,78 @@
+# Branching (showing pages depending on user input)
```

Comment:
Branching is only mentioning in the title, and nowhere else in the doc. Also, maybe 'branching' could confuse users (like me!) who have entry-level Git knowledge and might think we mean a Git thing. Because we've used "sends"/"shows" in the rest of the doc, could a title like 'Sending users to a different page' work? Or 'Showing pages after user input'?

Reply:
I think it's useful to introduce the term - always referring to it as something like 'showing pages depending on user input' is a bit long. How about we use the term again in the opening paragraph and define it, possible with a (this is different to concept of 'branching' in Git), unless we can think of another term?

Reply:
I'm a bit wary of us coining our own terms, as that can create issues later, but failing all else defining the concept sounds good. If we differentiate it from Git, though, what about non-technical users who do not know what branching means in either the Git or kit senses? Do we want to introduce them to 2 new, identically named concepts (only 1 of which is relevant to the tutorial)?

My other concern about 'branching' is that it's figurative language, so won't necessarily be understood at first glance. 

Potential issue with the title's 'showing pages depending on user input': it could also suggest that the page itself is somehow reliant on user input for its existence. What about 'page redirects after user input'?